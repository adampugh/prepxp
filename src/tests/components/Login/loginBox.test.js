import React from "react";
import { shallow } from "enzyme";

import LoginBox from "../../../components/Login/loginBox";

describe("<LoginBox />", () => {
    it("should match snapshot", () => {
        const wrapper = shallow(<LoginBox />);
        expect(wrapper).toMatchSnapshot();
    });
});