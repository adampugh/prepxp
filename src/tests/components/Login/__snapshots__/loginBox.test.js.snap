// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LoginBox /> should match snapshot 1`] = `
<div>
  <div
    className="login__box"
  >
    <h2>
      Log in to your account
    </h2>
    <p
      className="error"
    />
    <div
      className="login__box__input"
    >
      <input
        placeholder="email"
        type="email"
      />
      <i
        className="fas fa-envelope"
      />
    </div>
    <div
      className="login__box__input"
    >
      <input
        placeholder="password"
        type="password"
      />
      <i
        className="fas fa-lock"
      />
    </div>
    <button
      className="btn"
      onClick={[Function]}
    >
      Log In
    </button>
  </div>
  <div
    className="login__bottomBox"
  >
    <p>
      New to PrepXP? 
      <span>
        Sign up
      </span>
    </p>
  </div>
</div>
`;
